rust/src/utils.rs

```diff
@@ -929,19 +929,7 @@ pub fn internal_get_implicit_input(
                 |acc, ref withdrawal_amt| acc.checked_add(&withdrawal_amt)
             )?,
     };
-    let certificate_refund = match &certs {
-        None => to_bignum(0),
-        Some(certs) => certs.0
-            .iter()
-            .try_fold(
-                to_bignum(0),
-                |acc, ref cert| match &cert.0 {
```

Comment:
aren't these going in the opposite directions (register vs retirement)? the old code does it for `PoolRetirement`/`StakeDeregistration` but the new code would use `PoolRegistration` and `StakeRegistration`.

Reply:
Good catch. Somehow I missed this